webcompat/views.py

```diff
@@ -302,13 +302,18 @@ def cssfixme():
 @app.errorhandler(500)
 def custom_error_handler(err):
     # log the exception stack trace
-    app.logger.exception("!!!")
-    if api_call(request):
-        return api_message(err.code)
-    return render_template(
-        'error.html',
-        error_code=err.code,
-        error_message=ERROR_DICT[err.code]), err.code
+    # (but don't bother for localhost because the
+    # Flask debugger is already enabled)
+    if not app.config['LOCALHOST']:
+        app.logger.exception("Exception thrown:")
```

Comment:
Personally I don't mind collecting errors, even if it's noisy (and maybe we never look at them). It can be useful to come back to when investigating other things.

But like I said elsewhere, we should consider using Sentry which would mean removing this at some point anyways.